bitnami/elasticsearch/templates/ingress.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.master.ingress.enabled }}
+apiVersion: {{ include "common.capabilities.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.master.ingress.annotations .Values.commonAnnotations }}
+  annotations:
+    {{- if .Values.master.ingress.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.master.ingress.annotations "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+spec:
+  rules:
+    {{- if .Values.master.ingress.hostname }}
+    - http:
+        paths:
+          - path: {{ .Values.master.ingress.path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ .Values.master.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
```

Comment:
shouldn't serviceName be `include "elasticsearch.master.fullname" .`

Reply:
This was an oversight on my part.